src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <DefineConstants>CORECLR</DefineConstants>
+    <IncludeInPackage>Microsoft.ML.AutoML</IncludeInPackage>
+  </PropertyGroup>
+
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
```

Comment:
I see you are using `ProjectReference` here, but `PackageReference` above in the .nupkgproj. This won't work because you will be compiling against the latest code, but the NuGet package is referencing old binaries. So any new methods you use from ML.NET won't be in the old binaries, and cause runtime errors.

Reply:
replaced with nupkgproj references, checked what nuspec looks like in the final product and looks right to me.  Interesting that we only had to take a direct dependency on lgbm but Daniel says that's just a coincidence since others are pulled in from other dependencies.

  <dependencies>
      <group targetFramework=".NETStandard2.0">
        <dependency id="Microsoft.ML.LightGBM" version="1.3.0-preview-28005-0" exclude="Build,Analyzers" />
        <dependency id="Microsoft.ML.Mkl.Components" version="1.3.0-preview-28005-0" exclude="Build,Analyzers" />
        <dependency id="Microsoft.ML" version="1.3.0-preview-28005-0" exclude="Build,Analyzers" />
      </group>
    </dependencies>

This is what it used to be:

    <dependencies>
      <group targetFramework=".NETStandard2.0">
        <dependency id="Microsoft.ML" version="1.0.0-preview" include="All" />
        <dependency id="Microsoft.ML.LightGBM" version="1.0.0-preview" include="All" />
        <dependency id="Microsoft.ML.Mkl.Components" version="1.0.0-preview" include="All" />
      </group>
    </dependencies>